clinica/iotools/converters/oasis3_to_bids/oasis3_to_bids.py

```diff
@@ -0,0 +1,165 @@
+# coding: utf8
+
+"""Convert OASIS dataset (http://www.oasis-brains.org/) to BIDS."""
+
+from clinica.iotools.abstract_converter import Converter
+from clinica.utils.stream import cprint
+
+
+class Oasis3ToBids(Converter):
+    def convert_clinical_data(self, clinical_data_dir, bids_dir):
+        """Convert the clinical data defined inside the clinical_specifications.xlx into BIDS.
+
+        Args:
+            clinical_data_dir: path to the folder with the original clinical data
+            bids_dir: path to the BIDS directory
+        """
+        from os import path
+        import os
+        import numpy as np
+        import clinica.iotools.bids_utils as bids
+        from clinica.utils.stream import cprint
+
+        cprint("Converting clinical data...")
+        bids_ids = bids.get_bids_subjs_list(bids_dir)
+
+        iotools_folder = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+        clinic_specs_path = path.join(iotools_folder, "data", "clinical_specifications")
+
+        # -- Creation of modality agnostic files --
+        bids.write_modality_agnostic_files("OASIS-3", bids_dir)
+
+        # --Create participants.tsv--
+        participants_df = bids.create_participants_df(
+            "OASIS3", clinic_specs_path, clinical_data_dir, bids_ids
+        )
+
+        # Replace the values of the diagnosis_bl column
+        participants_df["diagnosis_bl"].replace([0.0, np.nan], "CN", inplace=True)
+        participants_df["diagnosis_bl"].replace(
+            [0.5, 1.0, 1.5, 2.0], "AD", inplace=True
+        )
```

Comment:
Are we sure `diagnosis_bl` will always take this specific set of values? Is the general rule that a strictly positive value of `diagnosis_bl` corresponds to "AD" otherwise "CN"?

Reply:
I based my work on the OASIS1 code: clinica/iotools/converters/oasis_to_bids/oasis_to_bids.py

Reply:
That does not anwer my question. I have a feeling that any value over 0 should be "AD", otherwise "CN" and the code could be simplified to reflect this rule.

Reply:
@14thibea, do you know the origin of that piece of code?

Reply:
It's based on the CDR score, so it makes sense that anything strictly above 0 is coded `AD` otherwise `CN`. So it would be best to replace this code with something like `paticipants_df["diagnosis_bl"] = paticipants_df["diagnosis_bl"].fillna(0).apply(lambda x: "AD" if x > 0 else "CN")`. That should handle all cases including missing values.